packages/azure-http-specs/specs/azure/resource-manager/resources/mockapi.ts

```diff
@@ -170,16 +264,30 @@ Scenarios.Azure_ResourceManager_Resources_ExtensionsResources_get = passOnSucces
       status: 200,
       body: json(validResourceGroupExtensionsResource),
     },
+    handler: (req: MockRequest) =>
+      requestHandler(req, "get", RESOURCE_GROUP_SCOPE_URI, validResourceGroupExtensionsResource),
     kind: "MockApiDefinition",
   },
   {
-    uri: "/subscriptions/:subscriptionId/providers/Azure.ResourceManager.Resources/extensionsResources/:extensionName",
+    uri: getUri(RESOURCE_GROUP_SCOPE_URI.substring(1), EXTENSION_RESOURCE_NAME, "get"),
```

Comment:
Do we have to always duplicate two operations? 
If `uri: /${resourceUri}/providers/Azure.ResourceManager.Resources/extensionsResources`, what will `resourceUri` be when the request is `//subscriptions/${SUBSCRIPTION_ID_EXPECTED}/providers/Azure.ResourceManager.Resources/extensionsResources`?

Reply:
1. Although the operations are repeated, the· `resourceUri` in the Uri is different. For example, the `resourceUri` of the Subscription scope is: `/subscriptions/${SUBSCRIPTION_ID_EXPECTED}` and `subscriptions/${SUBSCRIPTION_ID_EXPECTED}`
2. The resourceUri is `/subscriptions/${SUBSCRIPTION_ID_EXPECTED}` be when the request is `//subscriptions/${SUBSCRIPTION_ID_EXPECTED}/providers/Azure.ResourceManager.Resources/extensionsResources`.